src/Page/Form.purs

```diff
@@ -125,6 +125,15 @@ component =
 
   renderInput :: Field ExprType -> H.ParentHTML Query (ChildQuery m) ChildSlot m
   renderInput (field) = case field.input of
+    Currency currency ->
+      Input.currency_
+        [ HP.id_ field.key
+        , HP.placeholder (print currency.placeholder)
+        , HP.value $ fold do
+          value <- getValue currency
+          _ <- toCents value
```

Comment:
Hmm not sure I understand this, why is this being discarded?

Reply:
Sorry about that. This line might give us a `Cents` value. But we don't actually want that value, we want the `print`ed representation of it–the line below–if it exists. Does that make sense?

If we took the `Cents` value and rendered it to a `String`, we'd be spreading the logic of how we want to render from `print` to `Lynx.Page.Form`. It seemed better to not open that can of worms.

More than happy to restructure this whole thing so it's clearer if you've any suggestions.

Reply:
I guess I just don't know why we're calling `toCents` at all if we're not using the result. Or because we're in a `Maybe` monad, are we just using this line to validate our value before rendering? That's pretty clever